CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.0.4 (2020-08-05)
+
+- Fixed the colorpicker component to not positions its overlay until all contents have fully rendered. [#62](https://github.com/blackbaud/skyux-colorpicker/pull/62)
```

Comment:
"to not position" the (s) is unnecessary.

Reply:
Updated to Denise's wording